packages/react-scripts/config/webpack.config.js

```diff
@@ -44,6 +44,9 @@ const shouldUseSourceMap = process.env.GENERATE_SOURCEMAP !== 'false';
 // makes for a smoother build process.
 const shouldInlineRuntimeChunk = process.env.INLINE_RUNTIME_CHUNK !== 'false';
 
+// If an .eslintrc is found at the root of the app, then use it
+const hasLocalEslintrc = path.existsSync(path.relative(paths.appSrc, '.eslintrc'));
```

Comment:
Should this not be `fs.existsSync()`?  https://nodejs.org/api/fs.html#fs_fs_existssync_path

Also, please allow for `.eslintrc.js` as well.

Reply:
oops, ok fixed

Reply:
Looking good, thanks!

Hope it gets accepted.  :heart: